api/handlers/user.go

```diff
@@ -19,6 +19,30 @@ type User struct {
 	DB databases.UserDatabase
 }
 
+/*
+This funciton is passed on into the search handlers via the api
+handler, so there is the ability to verify a user is in a given
+community, as I saw your TODO comment in there.
+*/
+func (u User) VerifyInCommunity() func(string, string) bool {
```

Comment:
kudos: I like this function, this will be helpful in the future when we want to verify names belong to a certain user, etc. Nice!

suggestion: lets bubble up the errors from this function so they don't get lost.

```suggestion
func (u User) VerifyInCommunity() func(string, string) (bool, error) {
```

Reply:
Also, lets add some tests around this function. You can see some examples here: https://github.com/Linesmerrill/police-cad-api/blob/main/api/handlers/user_test.go

Reply:
Need to create the tests for this function too. I will add that in the next commit along with the tests for the plate-search